shanghai/client.py

```diff
@@ -21,6 +21,13 @@ def __init__(self, host, port, queue: asyncio.Queue, ssl=False, loop=None):
     def sendline(self, line):
         self.writer.write(line.encode('utf-8') + b'\r\n')
 
+    def sendcmd(self, command, params=None):
+        params = [] if params is None else params
+        args = [command] + params
+        if ' ' in args[-1]:
```

Comment:
If there are spaces in other parameters we'll have problems, no? Also, can the command contain spaces at all?